corehq/apps/es/migrations/0001_add_tombstone_in_mappings.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.16 on 2022-12-05 11:30
+
+from django.db import migrations
+from corehq.pillows.mappings.app_mapping import APP_ES_ALIAS
+from corehq.pillows.mappings.case_mapping import CASE_ES_ALIAS
+from corehq.pillows.mappings.case_search_mapping import CASE_SEARCH_ALIAS
+from corehq.pillows.mappings.domain_mapping import DOMAIN_ES_ALIAS
+from corehq.pillows.mappings.group_mapping import GROUP_ES_ALIAS
+from corehq.pillows.mappings.sms_mapping import SMS_ES_ALIAS
+from corehq.pillows.mappings.user_mapping import USER_ES_ALIAS
+from corehq.pillows.mappings.xform_mapping import XFORM_ALIAS
+
+from corehq.util.django_migrations import update_es_mapping
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        update_es_mapping(CASE_SEARCH_ALIAS),
```

Comment:
I think this will apply the entire mapping changes and I believe we would not want that? Is there is anything built in HQ for this? I can also use the normal es API to update the specific part.
Looking for your suggestions @calellowitz @joxl

Reply:
>

Correct. The ticket I'm currently working ([SAAS-14003](https://dimagi-dev.atlassian.net/browse/SAAS-14003)) on will add this.